src/react/DeviceStatusPage/index.tsx

```diff
@@ -0,0 +1,471 @@
+                    isAddingToGroup: false,
+                  })
+                }
+              />
+              <IntrospectionCard device={device} />
+              <PreviousInterfacesCard device={device} />
+              {fullInterfaceList.length > 0 && <ExchangedBytesCard device={device} />}
+              <DeviceStatusEventsCard device={device} />
+              <DeviceLiveEventsCard astarte={astarte} deviceId={device.id} />
+            </Row>
+          );
+        }}
+      </WaitForData>
+      {}
+      {activeModal && isWipeCredentialsModal(activeModal) && (
+        <ConfirmModal
+          title="Wipe Device Credentials"
+          confirmLabel="Wipe credentials secret"
+          confirmVariant="danger"
+          onCancel={dismissModal}
+          onConfirm={() => {
+            setActiveModal({ ...activeModal, isWipingCredentials: true });
+            wipeDeviceCredentials();
+          }}
+          isConfirming={activeModal.isWipingCredentials}
+        >
+          <p>
+            This will remove the current device credential secret from Astarte, forcing the device
+            to register again and store its new credentials secret. Continue?
+          </p>
+        </ConfirmModal>
+      )}
+      {activeModal && isAddToGroupModal(activeModal) && (
+        <AddToGroupModal
+          onCancel={dismissModal}
+          onConfirm={(groupName) => {
+            setActiveModal({ ...activeModal, isAddingToGroup: true });
+            addDeviceToGroup(groupName);
+          }}
+          groups={unjoinedGroups}
+          isAddingToGroup={activeModal.isAddingToGroup}
+        />
+      )}
+      {activeModal && isNewAliasModal(activeModal) && (
+        <NewAliasModal
+          onCancel={dismissModal}
+          onConfirm={({ key, value }) => {
+            setActiveModal({ ...activeModal, isAddingAlias: true });
+            handleAliasUpdate(key, value);
+          }}
+          isAddingAlias={activeModal.isAddingAlias}
+        />
+      )}
+      {activeModal && isEditAliasModal(activeModal) && (
+        <EditAliasModal
+          onCancel={dismissModal}
+          onConfirm={({ value }) => {
+            setActiveModal({ ...activeModal, isUpdatingAlias: true });
+            handleAliasUpdate(activeModal.targetAlias, value);
+          }}
+          targetAlias={activeModal.targetAlias}
+          isUpdatingAlias={activeModal.isUpdatingAlias}
+        />
+      )}
+      {activeModal && isDeleteAliasModal(activeModal) && (
+        <ConfirmModal
+          title="Delete Alias"
+          confirmLabel="Delete"
+          confirmVariant="danger"
+          onCancel={dismissModal}
+          onConfirm={() => {
+            setActiveModal({ ...activeModal, isDeletingAlias: true });
+            handleAliasDeletion(activeModal.aliasKey);
+          }}
+          isConfirming={activeModal.isDeletingAlias}
+        >
+          <p>{`Delete alias "${activeModal.aliasValue}"?`}</p>
```

Comment:
I see we had a pattern in other Delete modals to highlight the object of deletion with bold text:
`Delete alias <b>${activeModal.aliasValue}</b>?`
Do you think it would be fine to keep the same approach here?
Same for DeleteMetadataModal below

Reply:
for now i just wanted to copy the exact same modal we used before.
Changing the modal may also require changes on the tests.

Some modals use bold, some other wrap the word in double quotes ( " ), we should totally standardize those to use a common pattern

Reply:
Agreed